devel/ansible/roles/bodhi/tasks/main.yml

```diff
@@ -144,6 +144,10 @@
   args:
       creates: /home/vagrant/bodhi/development.ini
 
+- name: Change development.ini to make bodhi use the staging infrastructure
+  import_tasks: configure_stg.yml
+  when: fas is defined
```

Comment:
Using the `fas` variable for that seems a bit non-intuitive, I think it'd be better if we defined a `staging` variable or a `env` variable and set it to `staging`, and use that here.

Reply:
the fas variable is what is passed into ansible from vagrant (the changes to the vagrantfile now prompts the user for the fas username for the confgiure_stg.yml to add to the config).

I'll pass in the use_staging_infra boolean too, change the fas variable to `staging_fas` and change the when line to 

    when: (staging_fas is defined) and use_staging_infra

for extra clarity here.

does that sound ok?

Reply:
Yes, thanks :-)